apps/tudo/src/style/prosemirror.css

```
@@ -0,0 +1,62 @@
+  border-left: 1px solid #0d0d0d;
+  border-right: 1px solid #0d0d0d;
+  margin-left: -1px;
+  margin-right: -1px;
+  pointer-events: auto;
+  position: relative;
+  word-break: normal;
+}
+
+.collaboration-carets__label {
+  border-radius: 6px 6px 6px 0;
+  color: #ffffff;
```

Stylelint (17.2.0)

[error] Expected "#ffffff" to be "#fff" (color-hex-length)

(color-hex-length)